x/interchainstaking/keeper/callbacks_test.go

```diff
@@ -1607,5 +1612,81 @@ func (suite *KeeperTestSuite) TestDelegationAccountBalanceCallback() {
+		}
+
+		var proofBz []byte
+		for i := 0; i < len(proofMap); i++ {
+			key := fmt.Sprintf("%d", i)
+			proofBz = append(proofBz, byte(proofMap[key]))
+		}
+
```

Comment:
can you just dump base64 encoded proofbz here and then use the b64 value for the fixture. The above seems inefficient?

Reply:
I intend to encode this after I gather all infos I need.